ansible/roles/nginx/templates/nginx.conf.j2

```diff
@@ -17,13 +17,14 @@ http {
     access_log /logs/nginx_access.log combined-upstream;
 
     server {
+        listen 80;
```

Comment:
That's cool or just adjust the docs so it doesn't sound like it should just work, Line 82:
https://github.com/openwhisk/openwhisk/blob/03e041c416e19283cf1e1f2b58da85687c0c8dd5/tools/vagrant/README.md#L82